models/deepseek/v4/moe_ep.py

```
@@ -0,0 +1,1144 @@
+    routed_w3_i8_list = []
+    routed_w3_s_list = []
+    routed_w2_i8_list = []
+    routed_w2_s_list = []
+    for _ in range(N_RANKS):
+        w1_bf16 = (torch.randn(N_LOCAL, MOE_INTER, D) / D ** 0.5).to(torch.bfloat16)
+        w3_bf16 = (torch.randn(N_LOCAL, MOE_INTER, D) / D ** 0.5).to(torch.bfloat16)
+        w2_bf16 = (torch.randn(N_LOCAL, D, MOE_INTER) / MOE_INTER ** 0.5).to(torch.bfloat16)
+        w1_i8, w1_s = _quant_w_per_channel(w1_bf16)
+        w3_i8, w3_s = _quant_w_per_channel(w3_bf16)
+        w2_i8, w2_s = _quant_w_per_channel(w2_bf16)
+        routed_w1_i8_list.append(w1_i8); routed_w1_s_list.append(w1_s)
```

Ruff (0.15.14)

[error] Multiple statements on one line (semicolon)

(E702)